testcases/DeviceTreeValidation.py

```diff
@@ -236,24 +236,124 @@ def validate_pstate_properties(self):
                             "Non monotonocity observed for pstate ids")
 
     def check_dt_matches(self):
+        # allows the ability to filter for debug
+        # skip the parent(s) in the hierarchy
+        # leave as None to get checked
+        # setting to anything other than None will ignore
+        ignore_dict = {"/proc/device-tree/ibm,opal/"              : None,
+                       "/proc/device-tree/ibm,opal/sensor-groups" : None,
+                       "/proc/device-tree/ibm,opal/sensors"       : None,
+                       "/proc/device-tree/ibm,opal/power-mgt"     : None,
```

Comment:
out of interest, why are these nodes needing to be ignored?

Reply:
> why are these nodes needing to be ignored?

While debugging it was helpful to tune down the diffs to help visually identify if any nodes were truly not applicable on skiroot or host.  The code is meant for development usage and normally not seen or used.